API/ev3_sensor.c

```diff
@@ -372,6 +395,17 @@ int ReadSensor(int sensorPort)
 				return (-1)*(((help>>4) & 0xFF)*10 + ((help & 0xF) * 10 / 15)) * 9/5 + 320;
 			}
 			return (((help>>4) & 0xFF)*10 + ((help & 0xF) * 10 / 15)) * 9/5 + 320;
+		//New for HiTechnic IRSEEKER V2
+		case HT_DIR_DC:
+			return *((DATA16*)data);
```

Comment:
Leave these out, so it drops to the final `return *((DATA16*)data);` instead?

Reply:
Sounds good

Reply:
On second thought, it doesn't matter much. I incorporated the other changes, rebase and pushed it to the repository. Thanks.